front/src/common/redux/groups/groups.saga.js

```diff
@@ -0,0 +1,45 @@
+import { put, call, all, takeEvery } from 'redux-saga/effects';
+import { updateGroup, loadGroupsData, deleteStudent, getOneGroup } from './groups.api';
+import * as types from './types';
+
+export function* updateSaga(action) {
```

Comment:
Please, remove all exports here except export for `groupSaga`

Reply:
done